app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt

```diff
@@ -75,6 +78,8 @@ class HomeScreenRobot {
 
     fun verifyExistingTabList() = assertExistingTabList()
 
+    fun verifyTabTitle(expectedTitle: String) = assertTabTitle(expectedTitle)
+
     private fun scrollToElementByText(text: String): UiScrollable {
```

Comment:
you can instead use the helper method that's already defined here, rather than create a new one :)